jsons/parameters.json

```diff
@@ -15897,5 +16149,840 @@
+    "value_type": "int"
+  },
+  {
+    "_id": 394,
+    "default_value": 0,
+    "description": "This parameter specifies how theoretical fragment ion peaks are represented.",
+    "key_translations": {
+      "comet_style_1": "theoretical_fragment_ions",
+      "ursgal_style_1": "theoretical_fragment_ions"
+    },
+    "name": "theoretical_fragment_ions",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 395,
+    "default_value": [
+      600.0,
+      8000.0
+    ],
+    "description": "Defines the mass range of peptides to search (based on MH+ or the singly protonated mass).",
+    "key_translations": {
+      "comet_style_1": "digest_mass_range",
+      "ursgal_style_1": "digest_mass_range"
+    },
+    "name": "digest_mass_range",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "list"
+  },
+  {
+    "_id": 396,
+    "default_value": 20,
+    "description": "This parameter defines the maximum number of proteins (identifiers/accessions) to report. If set to -1, there will be no limit on the number of reported additional proteins.",
+    "key_translations": {
+      "comet_style_1": "max_duplicate_proteins",
+      "ursgal_style_1": "max_duplicate_proteins"
+    },
+    "name": "max_duplicate_proteins",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 397,
+    "default_value": 0,
+    "description": "This parameter sets the maximum indexed database search run time for a scan/query.",
+    "key_translations": {
+      "comet_style_1": "max_index_runtime",
+      "ursgal_style_1": "max_index_runtime"
+    },
+    "name": "max_index_runtime",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 398,
+    "default_value": 100,
+    "description": "This parameter controls the number of peptide search results that are stored internally. Depending on what post-processing tools are used, one may want to set this to the same value as num_output_lines (id193).",
+    "key_translations": {
+      "comet_style_1": "num_results",
+      "ursgal_style_1": "num_results"
+    },
+    "name": "num_results",
+    "tag": [],
+    "triggers_rerun": false,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 399,
+    "default_value": 0,
+    "description": "This parameter is used to search nucleotide sequence databases. It controls how the nucleotides are translated specifically which sets of reading frames are translated. Valid values are 0 through 9.",
+    "key_translations": {
+      "comet_style_1": "nucleotide_reading_frame",
+      "ursgal_style_1": "nucleotide_reading_frame"
+    },
+    "name": "nucleotide_reading_frame",
+    "tag": [],
+    "triggers_rerun": true,
+    "value_translations": {},
+    "value_type": "int"
+  },
+  {
+    "_id": 400,
```

Comment:
Not needed, in my opinion (if yes, indicate that it's comet specific)

Reply:
I actually messed it up a bit. The comet version implemented in ursgal2 atm is 2020.01.04, whereas this parameter actually is only introduced in 2021.01... thus for now I removed it and will introduce it one day if we update the comet version. It is used to match to old comet outputs... though only relevant for SQT output files.